Mustachio/TokenExpander.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Mustachio
+{
+    public enum Precedence
+    {
+        Low, Medium, High
+    }
+
+    public class TokenExpander
+    {
+        /// <summary>
+        /// RegEx used to identify the custom token.
+        /// </summary>
+        public string RegEx { get; set; }
+
+        /// <summary>
+        /// Low precedence expanders will be evaluated after all Mustache syntax tokens (e.g.: "each" blocks, groups, etc.).
+        /// Medium precedence expanders will be evaluated after "each" blocks and groups, but before unescaped variables {{{ var }}} syntax
+        /// High precedence expanders will be evaluated before all Mustache syntax tokens.
```

Comment:
Include that the order of the expanders in the passed in the array will be honored when applying them if more granularity is required.